File: src/main/java/com/senderman/stickersorterbot/StickersorterbotApplication.java
package com.senderman.stickersorterbot;

import com.annimon.tgbotsmodule.BotModule;
import com.annimon.tgbotsmodule.Runner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.telegram.telegrambots.ApiContextInitializer;

import java.util.List;

@SpringBootApplication
public class StickersorterbotApplication {

    public static void main(String[] args) {
        ApiContextInitializer.init();
        var context = SpringApplication.run(StickersorterbotApplication.class, args);
        var botModule = context.getBean(BotModule.class);
        Runner.run("", List.of(botModule));
    }

}

File: src/main/kotlin/com/senderman/stickersorterbot/StickerSorterBotApplication.kt
package com.senderman.stickersorterbot

import com.annimon.tgbotsmodule.Runner
import com.senderman.stickersorterbot.bot.SorterBot
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.telegram.telegrambots.ApiContextInitializer

@SpringBootApplication
class StickerSorterBotApplication

fun main(args: Array<String>) {
    ApiContextInitializer.init()
    val context = runApplication<StickerSorterBotApplication>(*args)
    Runner.run("", mutableListOf(context.getBean(SorterBot::class.java)))
}
